include/string_utils.h

```diff
@@ -217,6 +217,8 @@ inline bool is_empty(const char* str) noexcept
 // case-insensitive comparisons
 bool ciequals(const char a, const char b);
 
+bool case_insensitive_equals(const std::string& a, const std::string& b);
+
 // case-insensitive comparison for combinations of
 // const char *, const std::string&, and const string_view
 template <typename T1, typename T2>
```

Comment:
Maybe the new function isn't needed? Noticed the comment below is for insensitive string equality `iequal`, and used throughout the code base.

Reply:
Thanks, I overlooked that function somehow 😄 

I ended up with a much better solution using [std::filesystem::canonical](https://en.cppreference.com/w/cpp/filesystem/canonical) (and [std::filesystem::equivalent](https://en.cppreference.com/w/cpp/filesystem/equivalent) is another option). These use the actual underlying filesystem for name resolution and comparisons. Changes incoming soon!

Reply:
Looks great; didn't know about these filesystem "equivalency" calls.

Reply:
Yeah, they're great! Pushed new version.